packs/validator-0.0.40/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: validator
+description: Monitor results created by validator plugins and upload them to a configurable sink
```

Comment:
We don't need this file on the pack level. It is already part of the `charts/validation-0.0.40.tgz`